handlers.go

```diff
@@ -132,6 +132,14 @@ func isUserPath(username, path string) bool {
 	return len(path) == len(prefix) || path[len(prefix)] == '/'
 }
 
+func isMetricsPath(path string) bool {
+	prefix := "/metrics"
+	if !strings.HasPrefix(path, prefix) {
+		return false
+	}
+	return path == "/metrics"
```

Comment:
Sorry, this PR fell off my radar. Are the lines 136-139 necessary? I think they can be removed without changing functionality at all. What are your thoughts?

Reply:
@galexrt I don't really know how the metrics endpoints work, but couldn't this function just be `return strings.HasPrefix(path, "/metrics")`?

Reply:
The metrics endpoint is just a single path, `/metrics`, so the `==` is appropriate. :0

Reply:
I'll change it.